deployment/admission-webhook-datree.yaml

```diff
@@ -58,7 +58,7 @@ spec:
             requests:
               cpu: 100m
               memory: 256Mi
-          image: datree/admission-webhook:0.1.8
```

Comment:
I think it needs to be reverted, no?

Reply:
this field is being re-written by the ci - when pushing to main it gets the commit hash code, 
and when we run the workflow it will will update the file with the values from the prod yaml

Reply:
Check my last comment on Roys comment haha